ui/lib/open-api-explorer/addon/templates/components/swagger-ui.hbs

```diff
@@ -10,8 +10,11 @@
   <ToolbarFilters>
     <div class="field is-marginless">
       <p class="control has-icons-left">
+        <label for="swagger-result-filter" class="sr-only">Filter operations by path</label>
         <input
-          oninput={{queue (action "updateFilter") (action "proxyEvent")}}
```

Comment:
As an aside - I wonder if we should check for other `queue` actions in the codebase - was this causing the bug?

Reply:
Not by virtue of using the queue, but the proxyEvent was causing the input to lose focus, which we only want to happen once the user is done typing